package.json

```
@@ -1,48 +1,47 @@
 {
   "name": "@electron/fuses",
   "version": "0.0.0-development",
-  "main": "dist/index.js",
+  "type": "module",
+  "exports": "./dist/index.js",
   "license": "MIT",
   "bin": {
     "electron-fuses": "dist/bin.js"
   },
   "engines": {
-    "node": "^14.15.0 || ^16.10.0 || >=18.0.0"
+    "node": ">=22.12.0"
   },
   "publishConfig": {
     "provenance": true
   },
   "scripts": {
     "build": "tsc",
-    "test": "jest && tsc",
+    "build:docs": "npx typedoc src/index.ts",
+    "test": "vitest run && tsc",
```

Should we use `--noEmit` to avoid building files on the `test` command?

```suggestion
    "test": "vitest run && tsc --noEmit",
```